functions/export-csv.js

```diff
@@ -51,6 +52,12 @@ async function exportCsv(req, res) {
   });
 
   res.type("text/csv");
+  res.setHeader(
+    "Content-Disposition",
+    "attachment; filename=" +
+      layerName.replace(/[^a-z0-9]/gi, "-").toLowerCase() +
```

Comment:
LG! Please assign this "filenamization" operation to it's own function for better readability and testability. Thanks!